test/unit/client/test_client.py

```diff
@@ -61,6 +62,11 @@ def folder_id():
 
 
 @pytest.fixture(scope='module')
+def marker_id():
+    return 'marker_1'
+
+
+@pytest.fixture(scope='module')
```

Comment:
This `user_response` object was pre-existing, git just reports me as adding it. I added the lines above, with the @pytext.fixture from there. So for the comment on scope below, I followed suit from this line you're commenting on here. Additionally, I believe most of these fixtures are used in a read-only sense, so reuse of them in the tests is inconsequential.